Jenkinsfile

```diff
@@ -160,7 +160,6 @@ pipeline {
                     }
                     environment {
                         NEAR_WALLET_ENV = 'testnet_STAGING'
-                        REACT_APP_ACCOUNT_HELPER_URL = 'https://preflight-api.kitwallet.app'
```

Comment:
I'd prefer to continue bundling the MNW environments to use the AWS infrastructure, so I'll remove the other two instances when enabling this in `mainnet` environments. It gets tricky when all environments are in flux 😑